### ISSUE TITLE
remove social sharing

### ISSUE THREAD

**Issue by morninglucky** [nickbalestra/kactus#37]
I was just wondering how does one remove the social sharing links on posts? The Tweet and Facebook share buttons. I kind of want to remove them as I don't see anyone sharing my personal blog posts.

Can this happen?
Thanks!

**Comment by cjekel** [comment]
1. Remove share.html
2. Remove sharing section in https://github.com/nickbalestra/kactus/blob/0a02351167944404c33f3faabb23c3b8ec9fd29c/_layouts/post.html post.html lines 24-26
3. Change the link of your avatar to something other than twitter in https://github.com/nickbalestra/kactus/blob/0a02351167944404c33f3faabb23c3b8ec9fd29c/_layouts/post.html post.html line 16
4. Remove twitter from https://github.com/nickbalestra/kactus/blob/master/_config.yml _config.yml

It would be nice to have a True False option for this in the future... Like what is done with the disqus comments...

**Comment by morninglucky** [comment]
I completely agree. If I knew more about coding I would jump head first in helping this theme, I truly love it. Could I ask for help on one last thing? It's about the feed.xml file. My feed.xml is working on my server, but when you go to click on a blog-post it wants to link back to http://localhost:4000/2017/pioneers/
Did I do something wrong?

**Comment by cjekel** [comment]
Did you change feed.xml? The link should be <link>{{ site.url }}{{ post.url }}</link>
So I suspect your post is pioneers.md?

**Comment by morninglucky** [comment]
I don't remember changing the feed.xml code. Fact I don't think I wouldn't even know what to touch.
This is what my xml page says:
![screenshot 2017-12-11 19 47 18](https://user-images.githubusercontent.com/33889117/33863168-1f7dd6e0-deac-11e7-8fd7-7622105a60dd.png)

**Comment by morninglucky** [comment]
`---
layout: null
---
<?xml version="1.0" encoding="UTF-8"?>
<rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom" xmlns:dc="http://purl.org/dc/elements/1.1/">
  <channel>
    <title>{{ site.name | xml_escape }}</title>
    <description>{% if site.description %}{{ site.description | xml_escape }}{% endif %}</description>
    <link>{{ site.url }}</link>
    <atom:link href="{{ site.url }}/feed.xml" rel="self" type="application/rss+xml" />
    {% for post in site.posts limit:10 %}
      <item>
        <title>{{ post.title | xml_escape }}</title>
        {% if post.author.name %}
          <dc:creator>{{ post.author.name | xml_escape }}</dc:creator>
        {% endif %}
        {% if post.excerpt %}
          <description>{{ post.excerpt | xml_escape }}</description>
        {% else %}
          <description>{{ post.content | xml_escape }}</description>
        {% endif %}
        <pubDate>{{ post.date | date: "%a, %d %b %Y %H:%M:%S %z" }}</pubDate>
        <link>{{ site.url }}{{ post.url }}</link>
        <guid isPermaLink="true">{{ site.url }}{{ post.url }}</guid>
      </item>
    {% endfor %}
  </channel>
</rss>`

**Comment by morninglucky** [comment]
I feel like such a block head, when I got into the feed.xml in the site folder the code shows this - 
![screenshot 2017-12-11 19 49 10](https://user-images.githubusercontent.com/33889117/33863261-7591bdda-deac-11e7-91c9-48ae9ce3e00b.png)

How did it switch to local? If I delete the file will the site create a new one that works?

**Comment by cjekel** [comment]
I don't understand what is wrong.

If you are deploying the serve locally, then the link back to localhost is correct. 

If you are running a live site. Then just make sure the URLs in _config are set properly. The theme will do the rest.

**Comment by morninglucky** [comment]
Where can I check to see if: {{ site.url }}{{ post.url }} is right?

**Comment by morninglucky** [comment]
Ok so it's working now, seems I wasn't exporting the site using - Jekyll build --watch.
*smacks head*
